docs/security-considerations.rst

```diff
@@ -181,13 +181,14 @@ Sending and Receiving Ether
      contract. Again, the best practice here is to use a :ref:`"withdraw"
      pattern instead of a "send" pattern <withdrawal_pattern>`.
 
-Callstack Depth
-===============
+Call stack Depth
+================
 
 External function calls can fail any time because they exceed the maximum
 call stack of 1024. In such situations, Solidity throws an exception.
 Malicious actors might be able to force the call stack to a high value
-before they interact with your contract.
+before they interact with your contract, by controlling the number of calls
+and local function variables on the stack.
```

Comment:
```suggestion
before they interact with your contract. Note that, since [Tangerine Whistle](https://eips.ethereum.org/EIPS/eip-608) hardfork, the [63/64 rule](https://eips.ethereum.org/EIPS/eip-150) makes call stack depth attack impractical. Also note that the call stack and the expression stack are unrelated, even though both have a size limit of 1024 stack slots.
```